static/css/jira-configuration.css

```diff
@@ -156,6 +156,14 @@
   padding: 0.2rem;
 }
 
+/* Hit box */
+.jiraConfiguration__syncWarning:before {
+  content: '';
+  width: 3rem;
+  height: 3rem;
```

Comment:
Not a fan of having set height and width as it can affect ux if the warning gets bigger or it might go over another components hitbox nearby.

Reply:
Just noticed we're using a lot of mixed measurements in this css file. I think we're going to update this to align with what we decided on for the standard of the Jenkins app but right now we're using:
- px for borders
- em for margins and padding (so the margins and padding are relative to the parent element)
- rem for font (so it's relative to the root unit of measurement of the page)

Reply:
@mboudreau  - Changed it to a delightful 1.25em of **padding** instead of fixed size.
@rachellerathbone - replaced filthy **rem** with **em**.

Reply:
![image](https://user-images.githubusercontent.com/573668/172523916-ae1d2d43-907f-40f4-932a-b3bd60f56709.png)